src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -350,16 +365,16 @@ public void clickDeleteInstructorLinkAndCancel(int instrNum) {
     }
 
     public WebElement getNameField(int instrNum) {
-        return browser.driver.findElement(By.id("instructorname" + instrNum));
+        return browser.driver.findElement(By.id(Const.ParamsNames.INSTRUCTOR_NAME + instrNum));
```

Comment:
I think can just use the `@FindBy` defined at the top instead of `findElement()` againt